skpro/distributions/exponential.py

```diff
@@ -0,0 +1,74 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Exponential probability distribution."""
+
+__author__ = ["ShreeshaM07"]
+
+import pandas as pd
+from scipy.stats import expon, rv_continuous
+
+from skpro.distributions.adapters.scipy import _ScipyAdapter
+
+
+class Exponential(_ScipyAdapter):
+    r"""Exponential Distribution.
+
+    .. math::
+
+        f(x) = \exp(-x)
+
+    for :math:`x \ge 0`
+
+    Parameter
+    ---------
+    mu : float or array of float (1D or 2D)
+        mean of the distribution
+    scale : float or array of float (1D or 2D), must be positive
+        scale parameter of the distribution, same as standard deviation / sqrt(2)
+    index : pd.Index, optional, default = RangeIndex
+    columns : pd.Index, optional, default = RangeIndex
+    """
+
+    _tags = {
+        "capabilities:approx": ["ppf", "energy", "pdfnorm"],
+        "capabilities:exact": ["mean", "var", "pdf", "log_pdf", "cdf"],
+        "distr:measuretype": "discrete",
+        "broadcast_init": "on",
+    }
+
+    def __init__(self, mu, scale, index=None, columns=None):
+        self.mu = mu
+        self.scale = scale
+
+        super().__init__(index=index, columns=columns)
+
+    def _get_scipy_object(self) -> rv_continuous:
+        return expon
+
+    def _get_scipy_param(self) -> dict:
```

Comment:
return type here is a `dict`, but then you are returning a `tuple` composed of a `list` and a `dict`

Reply:
Indeed the return type must be changed.